doc/classes/Image.xml

```diff
@@ -166,52 +175,56 @@
 			<argument index="0" name="color" type="Color">
 			</argument>
 			<description>
-				Fills an [code]Image[/code] with a specified [Color]
+				Fills this [code]Image[/code] with a specified [Color].
 			</description>
 		</method>
 		<method name="fix_alpha_edges">
 			<return type="void">
 			</return>
 			<description>
+				Blends the pixels with low alpha values with colors of nearby pixels.
```

Comment:
- Two "with"s in a row is confusing. "Blends" implies color.

Blends low-alpha pixels with nearby pixels.